utils/update_tools/Makefile

```diff
@@ -0,0 +1,21 @@
+# Temporary repositories to perform master+wip update
+SHELL:=/bin/bash
+
+VTR=vtr
+YOSYS=yosys
+
+clean:
+	rm -rf ${VTR} ${YOSYS}
+
+update-vtr:
+	./update_tools.py git@github.com:SymbiFlow/vtr-verilog-to-routing.git ${VTR}
+	# Testing that VTR is correctly integrated in SymbiFlow
+	pushd ${VTR};                                   \
+	make -j`nproc`;                                 \
+	export VTR=`pwd`/build/vpr/vpr;                 \
+	export GENFASM=`pwd`/build/utils/fasm/genfasm;  \
+	popd                                            \
+	make all_xc7 && make all_ice40 && make all_v2x
```

Comment:
Should probably support parallel build here (e.g. '-jN')

Reply:
Done